cmd/cli/cmd/manifest/generate/implementations/terraform.go

```diff
@@ -3,9 +3,10 @@ package implementations
 import (
```

Comment:
Please remove the `alpha` command and `s/manifest-gen/generate` from the **Example** section in line 26,29,32